test/helpers/quictestlib.c

```diff
@@ -77,10 +77,17 @@ static void handshake_finish(void *arg);
 
 static int using_fake_time = 0;
 static OSSL_TIME fake_now;
+static CRYPTO_RWLOCK *fake_now_lock = NULL;
 
 static OSSL_TIME fake_now_cb(void *arg)
 {
-    return fake_now;
+    OSSL_TIME ret;
+    if (!CRYPTO_THREAD_read_lock(fake_now_lock))
+        ret.t = 0;
```

Comment:
`= ossl_time_zero()` (don't access the internal structure of OSSL_TIME directly)

Reply:
The reason being that the internal format is subject to change.